subworkflows/nf-core/vcf_extract_relate_somalier/meta.yml

```diff
@@ -19,8 +19,10 @@ input:
   - ch_vcfs:
       type: file(s)
       description: |
-        The input VCFs to perform the stats on, optionally with their indices
-        Structure: [ meta, vcf, (tbi|[]) ]
+        The input VCFs to perform the stats on, optionally with their indices.
+        This channel can also contain the number of samples in the same family/group
+        to check relatedness. This is adviced to add as it can improve the efficiency of your pipeline
```

Comment:
```suggestion
        to check relatedness. This is advised to add as it can improve the efficiency of your pipeline
```

Reply:
You imply the `count` is optional I think? Maybe specify what it can look like when being used and when not?

Reply:
Yes it's optional indeed, thanks for the suggestion I'll clarify it